exercises/error-handling/.meta/src/reference/java/ErrorHandling.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Optional;
+
+class ErrorHandling {
+
+    public void handleErrorByThrowingException() throws Exception {
+        throw new Exception();
+    }
+
+    public void handleErrorByThrowingExceptionWithDetailMessage(String message) throws Exception {
```

Comment:
`public` methods could be changed to package-private 🙂

Reply:
True 🙂